tests/sample_data/pulse_consumer/github_push.json

```diff
@@ -1,16 +1,263 @@
+            "username": "bors[bot]"
+          },
+          "committer": {
+            "name": "GitHub",
+            "email": "noreply@github.com",
+            "username": "web-flow"
+          },
+          "added": [],
+          "removed": [],
+          "modified": [
+            "components/service/experiments/src/main/java/mozilla/components/service/experiments/ExperimentsUpdater.kt",
+            "components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt"
+          ]
+        }
+      ],
+      "head_commit": {
+        "id": "5fdb785b28b356f50fc1d9cb180d401bb03fc1f1",
+        "tree_id": "410cf16b218c00bf743585d1d9cce5d89f418b97",
+        "distinct": true,
+        "message": "[ci skip][skip ci][skip netlify] -bors-staging-tmp-5835",
+        "timestamp": "2020-02-12T15:29:12Z",
+        "url": "https://github.com/mozilla-mobile/android-components/commit/5fdb785b28b356f50fc1d9cb180d401bb03fc1f1",
+        "author": {
+          "name": "bors[bot]",
+          "email": "26634292+bors[bot]@users.noreply.github.com",
+          "username": "bors[bot]"
+        },
+        "committer": {
+          "name": "GitHub",
+          "email": "noreply@github.com",
+          "username": "web-flow"
+        },
+        "added": [],
+        "removed": [],
+        "modified": [
+          "components/service/experiments/src/main/java/mozilla/components/service/experiments/ExperimentsUpdater.kt",
+          "components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt"
+        ]
+      }
+    },
+    "organization": "mozilla-mobile",
+    "details": {
+      "event.base.ref": "refs/heads/master",
+      "event.base.repo.name": "android-components",
+      "event.base.repo.url": "https://github.com/mozilla-mobile/android-components.git",
+      "event.base.sha": "7285afe57ae6207fdb5d6db45133dac2053b7820",
+      "event.base.user.login": "bors[bot]",
+      "event.head.ref": "refs/heads/master",
+      "event.head.repo.name": "android-components",
+      "event.head.repo.url": "https://github.com/mozilla-mobile/android-components.git",
+      "event.head.sha": "5fdb785b28b356f50fc1d9cb180d401bb03fc1f1",
+      "event.head.user.login": "bors[bot]",
+      "event.head.user.id": 26634292,
+      "event.type": "push",
+      "event.base.repo.branch": "master",
+      "event.head.repo.branch": "master",
+      "event.head.user.email": "bors@users.noreply.github.com"
+    },
+    "installationId": 5110595,
+    "tasks_for": "github-push",
+    "branch": "master",
+    "repository": "android-components",
+    "eventId": "6c722d22-4dac-11ea-8d13-04e94ccdb80b",
+    "version": 1
   },
-  "repository": "test_treeherder",
-  "version": 1
+  "exchange": "exchange/taskcluster-github/v1/push",
+  "routingKey": "primary.mozilla-mobile.android-components",
+  "redelivered": false,
+  "cc": []
```

Comment:
Do we need all of this for the tests?

Reply:
No we don't, however, it is better to keep the whole data structure in case in the future we want to inspect what other attributes could be useful for other purposes.

For instance, the attribute `timestamp` was available to us, however, it was nowhere documented in the Github APIs or Taskcluster's code. This feature *could* have been fixed a while back if the complete data structure had been explicitely somewhere.

Fixing this kind of bugs is not easy because not everything is documented and inspecting manually the data is what ultimately can help them be solved.

Reply:
ok, fair enough.